core/services/llo/keyring.go

```diff
@@ -0,0 +1,78 @@
+package llo
+
+import (
+	"fmt"
+
+	"github.com/smartcontractkit/libocr/offchainreporting2/types"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
+	ocrtypes "github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+
+	commontypes "github.com/smartcontractkit/chainlink-common/pkg/types"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+)
+
+type LLOOnchainKeyring ocr3types.OnchainKeyring[commontypes.LLOReportInfo]
```

Comment:
nit: i'd prefer that the common repo is changed to have an llo package in `pkg/types/llo` and then this can be ~

import llotypes `...common/pkg/types/llo`
llotypes.ReportInfo

Reply:
https://github.com/smartcontractkit/chainlink-common/pull/351

Reply:
https://github.com/smartcontractkit/chainlink-data-streams/pull/31